styleguide/overlay.scss

```diff
@@ -11,9 +11,11 @@ $overlay-margin: 10vh;
   height: 100%;
   left: 0;
   opacity: 0;
-  overflow: scroll;
+  overflow: auto;
+  -webkit-overflow-scrolling: touch;
   position: fixed;
   top: 0;
+  transform: translateZ(0); // create a new viewport context
```

Comment:
This will actually force most browsers to offload more to the GPU, and will create a new "viewport context". Normally creating a new viewport context is a _weird, unexpected behavior_ but it's fine when doing it on a fixed position element.